catlearn/fingerprint/adsorbate_fingerprint.py

```diff
@@ -6,21 +6,25 @@
 from .periodic_table_data import (get_mendeleev_params, n_outer,
                                   list_mendeleev_params,
                                   default_params, get_radius,
-                                  electronegativities)
+                                  electronegativities,
+                                  block2number)
 from .neighbor_matrix import connection_matrix
 import collections
 from .base import BaseGenerator
 
 
-block2number = {'s': 1,
```

Comment:
This was changed and moved over to `.periodic_table_data` recently, so this is a conflict.